lib/processors/lib/jaccard.py

```diff
@@ -0,0 +1,133 @@
+import random
+
+def jaccard_rec(current_answers):
```

Comment:
@mynameisfiber I designed it when we weren't sure how we were gathering responses. The possibility of having someone answer some, but not all, of the questions was one I decided to deal with. So, if there's missing entries, I fill them in. It turned out that the form we ended up using required all questions to be answered. I didn't change it, since I didn't think it added much processing time, and in the future, we might hand out surveys with randomized sets of questions.